dev/com.ibm.ws.org.apache.cxf.ws.security/override/META-INF/cxf/bus-extensions.txt

```diff
@@ -1 +0,0 @@
-org.apache.cxf.ws.security.cache.CacheCleanupListener::true:true
```

Comment:
These resources and nls files belong still in the 2.3 one.  So if you fix this back up to leave this project here with the updated bnd we will be good.